docs/reference/body-segmentation.md

```diff
@@ -170,18 +174,25 @@ const bodySegmentation = ml5.bodySegmentation(?modelName, ?options, ?callback);
   }
   ```
 
-  [More info on options for SelfieSegmentation model with mediaPipe runtime](https://github.com/tensorflow/tfjs-models/tree/master/body-segmentation/src/selfie_segmentation_mediapipe#create-a-detector).
+  Important Option:
+  - **maskType**: The type of mask to output. The options are:
+    - _background_: A mask of the background. The result is an image with transparent pixels on the background and black pixels on the person.
+    - _body_: A mask of the person. The result is an image with black pixels on the background and transparent pixels on the person.
+    - _parts_: **BodyPix** only. A mask of the body parts. The result is an image with white pixels on the background and various color pixels for each body part.
+
+    [More info on options for SelfieSegmentation model with mediaPipe runtime](https://github.com/tensorflow/tfjs-models/tree/master/body-segmentation/src/selfie_segmentation_mediapipe#create-a-detector).
 
-  [More info on options for SelfieSegmentation model with tfjs runtime](https://github.com/tensorflow/tfjs-models/tree/master/body-segmentation/src/selfie_segmentation_tfjs#create-a-detector).
+    [More info on options for SelfieSegmentation model with tfjs runtime](https://github.com/tensorflow/tfjs-models/tree/master/body-segmentation/src/selfie_segmentation_tfjs#create-a-detector).
 
   [More info on options for BodyPix model.](https://github.com/tensorflow/tfjs-models/blob/master/body-segmentation/src/body_pix/README.md#create-a-detector)
 
-- **callback(bodySegmentation, error)**: OPTIONAL. A function to run once the model has been loaded. Alternatively, call `ml5.bodySegmentation()` within the p5 `preload` function.
+- **callback(bodySegmentation)**: OPTIONAL. A function to run once the model has been loaded. Alternatively, call `ml5.bodySegmentation()` within the p5 `preload` function.
```

Comment:
I remembered we still have error, it's just we don't use it in example code. We could double check this after soft launch, feel free to create a Issue for this.

Reply:
Ok, I will consider putting it back with the `?` indicating that it is optional. It is actually quite important for debugging.